examples/accordion/css/accordion.css

```diff
@@ -60,11 +60,17 @@
 .Accordion-panel {
     margin: 0;
     padding: 1em 1.5em;
+    overflow: hidden;
+    transition: all 500ms;
 }
 
-/* For Edge bug https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/4806035/ */
 .Accordion-panel[hidden] {
-  display: none;
+  display: block;
+  height: 0;
+  padding-top: 0;
+  padding-bottom: 0;
+  border-top: 0;
+  visibility: hidden;
```

Comment:
Setting `visibility` to hidden here prevents the majority of the animation from happening since it causes the element to have no height, which in turn means that the height cannot transition to 0. The end result is that only the padding is actually transitioning.